internal/cache/controller_obj_cache.go

```diff
@@ -2471,7 +2471,7 @@ func (c *AviObjCache) AviObjOneVSCachePopulate(client *clients.AviClient, cloud
 			vs_parent_ref, foundParent := vs["vh_parent_vs_ref"]
 			var parentVSKey NamespaceName
 			if foundParent {
-				vs_uuid := ExtractUuidWithoutHash(vs_parent_ref.(string), "virtualservice-.*.")
+				vs_uuid := ExtractUuid(vs_parent_ref.(string), "virtualservice-.*.#")
```

Comment:
vs_uuid -> vsUUID We should be using camelCase instead of snake case in Go.
ExtractUuid -> ExtractUUID

Reply:
Done